manual/trees/index.md

```diff
@@ -190,13 +157,168 @@ The `TTree::Write()` method is needed to write the ROOT file header.
+   print(entry.branchName)
+{% endhighlight %}
+
+
+### Selecting a subset of branches to be read
+
+You can select or deselect branches from being read by `GetEntry()` by calling [`TTree::SetBranchStatus()`](https://root.cern/doc/master/classTTree.html#aeca53bcd4482de1d883d259df6663920).
+It is vividly recommended to only read the branches actually needed:
+`TTree` is optimized for exactly this use case, and most analyses will only need a fraction of the available branches.
+
+{% highlight C++ %}
+// Extract the tree as above.
+
+// Disable everything...
+tree->SetBranchStatus("*", false);
+// ...but the branch we need
+tree->SetBranchStatus("branchName", true);
+
+// Now proceed as above.
+int variable;
+tree->SetBranchAddress("branchName", &variable);
+for (int iEntry = 0; tree->LoadTree(iEntry) >= 0; ++iEntry) {
+   // Load the data for the given tree entry
+   tree->GetEntry(iEntry);
+
+   printf("%d\n", variable);
+}
+{% endhighlight %}
 
-`Autosave` gives you the option to save all branch buffers every n byte. It is recommended to use `Autosave` for large acquisitions. If the acquisition fails to complete, you can recover the ROOT file and all the contents since the last `Autosave`.
 
-- Use the [TTree::SetAutosave()](https://root.cern/doc/master/classTTree.html#a76259576b0094536ad084cde665c13a8){:target="_blank"} method to set the number of bytes between `Autosave`.
+### Selecting a subset of entries to be read
 
-You can also use [TTree::SetAutosave()](https://root.cern/doc/master/classTTree.html#a76259576b0094536ad084cde665c13a8){:target="_blank"} in the acquisition loop every n entry.
+To process only a selection of tree entries, you can use a {% include ref class="TEntryList" %}.
+First you inserting the tree entry numbers you want to process into the `TEntryList`.
+
+{% highlight Python %}
+entryList = ROOT.TEntryList("entryListName", "Title of the entry list")
+for entry in tree:
+   if entry.missingET < 100:
+      entryList.Enter(tree.GetReadEntry())
+myFile = ROOT.TFile.Open("entrylist.root", "RECREATE")
+myFile.WriteObject(entrylist)
+{% endhighlight %}
+
+You can then re-use the `TEntryList` in subsequent processing of the tree, skipping irrelevant entries.
+
+{% highlight Python %}
+myFile = ROOT.TFile.Open("entrylist.root")
+entrylist = myFile.entryListName
+tree.SetEntryList(entrylist)
+for entry in tree:
+   # all entries will have missingET < 100
+{% endhighlight %}
+
+## Appending `TTree`s as a `TChain`
+
+In high energy physics you always want as much data as possible.
+But it's not nice to deal with files of multiple terabytes.
+ROOT allows to to split data across multiple files, where you can then access the files' tree parts as one large tree.
+That's done through {% include ref class="TChain" %}, which inherits from {% include ref class="TTree" %}:
+it wants to know the name of the trees in the files (has to be the same throughout), and the file names, and will act as if it was a huge, continuous tree:
```

Comment:
```suggestion
it wants to know the name of the trees in the files, and the file names, and will act as if it was a huge, continuous tree:
```
> (has to be the same throughout),

TChain actually works properly with names change as long as it is specified when adding the file to the chain.

Reply:
Clarified